[Bug Report] Error Searching Manga with MangaDex Plugin

**Describe the bug**
When searching for manga using the Mangadex plugin, the following error is received:

`Search failed. Error: Error: Missing required fields in the Manga type`

**To Reproduce**
Steps to reproduce the behavior:
1. Navigate to Download > Plugins
2. Select MangaDex from the dropdown
3. Search for any manga title (sans punctuation, since that also throws an error)

**Expected behavior**
A search result should be returned by the search attempt

**Environment (please complete the following information):**

**Client**
- Firefox flatpak, ver 98.0.2
- Pop_OS! 21.10

**Server**
- Unraid 6.9.2 running Docker ver 20.10.5
- Mango ver 0.26.0

**Docker (if you are running Mango in a Docker container)**
version: '2'
services:
    mango:
        image: hkalexling/mango
        container_name: ""
        environment:
            - TZ=America/New_York
            - HOST_OS=Unraid
        volumes:
            - /mnt/disk2/dockerRoot/mango/appdata
            - /mnt/disk1/dockerRoot/mango/mangas

**Additional context**
Logged in stdout as well:
[91m[ERROR] [0m2022/04/03 20:05:43 | Missing required fields in the Manga type
[91m[ERROR] [0m2022/04/03 20:05:43 | Missing required fields in the Manga type

--- a/plugins/mangadex/index.js
+++ b/plugins/mangadex/index.js
@@ -54,9 +54,10 @@ function searchManga(query) {
 		mango.raise('Failed to search for manga.');
 
 	return JSON.stringify(manga.map(function(m) {
+		const titleAttr = m.attributes.title;
 		const ch = {
 			id: m.id,
-			title: m.attributes.title.en,
+			title: titleAttr.en || titleAttr['ja-ro'] || titleAttr[Object.keys(titleAttr)[0]]
 		};
 		for (i = 0; i < m.relationships.length; i++) {
 			const obj = m.relationships[i];

